configurecompiler.cmake

```diff
@@ -8,7 +8,7 @@ if(CMAKE_SYSTEM_NAME STREQUAL Linux)
     if(CLR_CROSS_COMPONENTS_BUILD)
         # CMAKE_HOST_SYSTEM_PROCESSOR returns the value of `uname -p` on host.
         if(CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL x86_64 OR CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL amd64)
-            if(CLR_CMAKE_TARGET_ARCH STREQUAL "arm")
+            if(CLR_CMAKE_TARGET_ARCH STREQUAL "arm" OR CLR_CMAKE_TARGET_ARCH STREQUAL "armel")
```

Comment:
Maybe, you will want to add "armv7l" as well if you consider softfp.

Ref: https://github.com/rpm-software-management/rpm/blob/7ffc4d17ffa8f87bd9107b5dc4d9e25daadd14ae/macros.in#L1119

Reply:
@myungjoo the CLR_CMAKE_TARGET_ARCH is generated by the build.sh and we never set it to `armv7l`.